### ISSUE TITLE
translation tool

### ISSUE THREAD

**Issue by SimonKoehl** [serlo-org/PlantBuddies#12]
Hi Botho, do you know http://crowdin.com/ ? We use it to translate serlo.org. As far as I know, it supports classic language files as well as an API for strings. May be, that is of help. It would make the translation of plant buddies into other languages as well as the update of the English version very easy.

**Comment by elbotho** [comment]
I agree we need a better solution in the long run!
Uncertainties:
- Is crowdin free for ngo's?
- we probably need a more complex backend ui anyway at add and edit plants and relations (translations being a small part of that)

what do you think?

**Comment by SimonKoehl** [comment]
Crowdin would be for free, You could just add a translation file to the serlo project: https://crowdin.com/project/serlo/de That way, other Serlo Translaters might automaticaly help to translate plant buddies.

I don't recommend to build a backend for translations. If we use crowdin, our translaters can discuss their translations and have other useful features

**Comment by elbotho** [comment]
@SimonKoehl I don't mean a translation backend, I mean a backend to edit the data in general

**Comment by SimonKoehl** [comment]
yeah, I understood, I just ment, that it might be useful to exclude translations from this backend never the less. 

Peace
